packages/admin/components/layout/Drawer/Drawer.tsx

```diff
@@ -31,13 +32,16 @@ const Drawer = ({
     if (onClose) onClose();
   };
 
+  usePreventBodyScroll(dialog.visible);
+
   return (
     <Styled.DialogBackdrop {...dialog}>
       <Styled.Dialog
         aria-labelledby={uidLabel}
         aria-describedby={uidDesc}
         hideOnClickOutside={hideOnClickOutside}
         {...dialog}
+        preventBodyScroll={false}
```

Comment:
@1aurend tiny nit - spread operators should be at the end of the list

Reply:
Oh, yep. Np!